decidim-assemblies/app/cells/decidim/assemblies/assembly_cell.rb

```diff
@@ -12,7 +12,12 @@ def show
       private
 
       def card_size
-        "decidim/assemblies/assembly_m"
+        case @options[:size]
+        when :g
+          "decidim/assemblies/assembly_g"
+        else
+          "decidim/assemblies/assembly_m"
```

Comment:
Add a comment because size M is deprecated?

Reply:
Done!